CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Unreleased
 ### Accessibility
 
 ### Bug Fixes Users Might Notice
+- Layout: The "Delete Account" confirmation page has a new layout consistent with other content pages. (#5857)
```

Comment:
I'd say this is an improvement/change even if it's small?

Reply:
>I'd say this is an improvement/change even if it's small?

Sure, makes sense to me too, since it's not really a bug fix. Was mostly following notes from the previous release as reference.